### PR TITLE
Publishable Package

### PR DESCRIPTION
**Pull request by RoyalSix**
## Description
Example usage
https://codesandbox.io/s/crt-vty71

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> `Fix [#1]()`

### CODE REVIEW

File: src/components/SideBar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+} from "./styled";
+
+function SideBar({
+  open, dismiss, href,
+}) {
+  const { config: { theme } } = useContext(ToastContext);
+  return (
+    <Container theme={theme.dismiss}>
+      {open && (
+        <>
```
Comment (rileylnapier) [inline src/components/SideBar/index.tsx]: don't need the fragment

File: src/components/Toast/index.tsx [hunk 2]
```diff
@@ -7,6 +7,7 @@ const Toast: React.FunctionComponent = () => {
   const { config } = useContext(ToastContext);
 
   const Transition = getTransition(config.transition);
+  console.log("config", config);
```
Comment (rileylnapier) [inline src/components/Toast/index.tsx]: remove

File: src/components/SideBar/index.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+import {
+  Container, Button,
+} from "./styled";
+
+function SideBar({
+  open, dismiss, href,
+}) {
+  const { config: { theme } } = useContext(ToastContext);
+  return (
+    <Container theme={theme.dismiss}>
```
Comment (RoyalSix) [inline src/components/SideBar/index.tsx]: not theme.dismiss

File: src/components/SideBar/index.tsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+      {open && (
+        <Button theme={theme.dismissButton} href={href} color="#9D3789"
+          onClick={open}>Details</Button>
+      )}
+      <Button theme={theme.dismissButton} onClick={dismiss}>Dismiss</Button>
```
Comment (RoyalSix) [inline src/components/SideBar/index.tsx]: theme.detials/dismiss

File: src/constants/index.ts [hunk 5]
```diff
@@ -0,0 +1 @@
+export const COURIER_CLIENT_HEADER = "X-COURIER-CLIENT-KEY";
```
Comment (RoyalSix) [inline src/constants/index.ts]: lowercase

File: src/transports/index.ts [hunk 6]
```diff
@@ -1,3 +1,2 @@
 export * from "./courier";
-export * from "./pusher";
 export * from "./base";
```
Comment (RoyalSix) [inline src/transports/index.ts:2]: rollup

**Review by rileylnapier — APPROVED** [review]

**Pull request merged**